### ISSUE TITLE
Sometimes Fails to Handle HE-AAC 

### ISSUE THREAD

**Issue by LeeThompson** [RandomEngy/VidCoder#554]
This is *probably* a Handbrake issue but I should run it past the VidCoder team first just in case.

I've had this happen on a few files with the audio track identified as "Unknown (HE-AAC) (2.0 ch)".   One oddity is Bitrate shows unknown in VidCoder's UI as well.

When transcoding with VidCoder 4.35 (x64), _the resultant file has no audio track at all._   it is the same if it's on passthrough or re-encode the AAC track.

(The source file does play audio in VLC.)

MediaInfo shows:
```
ID                                       : 1
Format                                   : AAC LC SBR
Format/Info                              : Advanced Audio Codec Low Complexity with Spectral Band Replication
Commercial name                          : HE-AAC
Format settings                          : Implicit
Codec ID                                 : A_AAC-2
Duration                                 : 22 min 3 s
Bit rate                                 : 127 kb/s
Channel(s)                               : 2 channels
Channel layout                           : L R
Sampling rate                            : 48.0 kHz
Frame rate                               : 23.438 FPS (2048 SPF)
Compression mode                         : Lossy
Stream size                              : 20.0 MiB (4%)
Default                                  : Yes
Forced                                   : No
```

VidCoder's log shows (audio related):
```
[22:57:19]  * audio track 1
[22:57:19]    + decoder: Unknown (HE-AAC) (2.0 ch) (track 1, id 0x0)
[22:57:19]      + samplerate: 48000 Hz
[22:57:19]    + mixdown: Stereo
[22:57:19]    + dither: none
[22:57:19]    + encoder: AAC (libavcodec)
[22:57:19]      + bitrate: 384 kbps, samplerate: 48000 Hz
...
[23:00:34] aac-decoder done: 0 frames, 0 decoder errors
```

**Comment by RandomEngy** [comment]
If you can get this file to me (or a sample that reproduces the problem) that will be helpful. I have been hearing reports of people having issues with "unknown bitrate" tracks but haven't gotten to see it myself.

**Comment by LeeThompson** [comment]
I can probably arrange that.

**Comment by RandomEngy** [comment]
Thanks, I got it. But I don't have any trouble encoding it with the audio. Can you send me your %appdata%\VidCoder-Beta\VidCoder.sqlite file and let me know what picker/encoder you're using? That will eliminate encoder settings as the culprit.

**Comment by RandomEngy** [comment]
Okay I did a bunch of testing and isolated the issue. When the subtitle is included, the audio tracks drop. This reproduces on the latest HandBrake nightly. I've opened an issue on them: https://github.com/HandBrake/HandBrake/issues/2103

**Comment by LeeThompson** [comment]
Awesome! Thanks for looking into that!

**Comment by RandomEngy** [comment]
HandBrake fixed it on their end. Should be resolved in 5.11 Beta.
